src/Misc/layoutbin/darwin.svc.sh.template

```diff
@@ -107,7 +98,7 @@ function status()
     fi
 
     echo
-    status_out=`sudo -u "${SUDO_USER}" launchctl list | grep "${SVC_NAME}"`
+    status_out=`launchctl list | grep "${SVC_NAME}"`
```

Comment:
I've read that the \` (backtick) syntax is "deprecated" and `$()` is preferred instead. I'm not sure exactly what deprecated means, I can't imagine it won't work. It may simply be because backticks are hard to distinguish.